### PR TITLE
Add details to the release process in devGuide

### PR DESCRIPTION
**Pull request by tlylt**
**What is the purpose of this pull request?**

- [x] Documentation update
- [ ] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] DevOps
- [ ] Improve developer experience
- [ ] Others, please explain:

**Overview of changes:**
Include what I observed during the release process to make certain points in the guide clearer.

**Anything you'd like to highlight/discuss:**
nil

**Testing instructions:**
https://deploy-preview-2168--markbind-master.netlify.app/devguide/projectmanagement#doing-a-release

**Proposed commit message: (wrap lines at 72 characters)**
Add details to the release process in devGuide

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [ ] Linked all related issues
- [x] No unrelated changes

### CODE REVIEW

File: docs/devGuide/projectManagement.md [hunk 1]
```diff
@@ -139,12 +146,14 @@ For general best practices, refer to the guide [_Working with PRs_ @SE-EDU](http
...
 
-   We do this for each MarkBind release (rather than require contributors to update the bundles for each affected PR), in order to reduce unnecessary merge conflicts. It also makes it easier for the maintainers to vet the changes.
+   * We do this for each MarkBind release (rather than require contributors to update the bundles for each affected PR), in order to reduce unnecessary merge conflicts. It also makes it easier for the maintainers to vet the changes.
+   * Typically, the bundle will be updated automatically in the expected test site's folders after running `npm run updatetest` in the previous step.
    </box>
 
 1. **Combine** the changes that you made in the last two steps with the [version commit](https://docs.npmjs.com/cli/version) generated by `lerna version`. To do so, amend the version commit and version tag by doing the following — take note to change `vA.B.C` to the new version.
 
    ```sh
+   $ git add .
```
Comment (jonahtanjz) [inline docs/devGuide/projectManagement.md:156]: I believe the command `git commit -a` below would already automatically stage the modified files? There shouldn't be any new untracked files created during release so there is no need to explicitly run `git add`?

**Review by jonahtanjz — COMMENTED** [review]
Thanks @tlylt for updating the release guide!

Just one comment, other than that the rest looks good 👍

File: docs/devGuide/projectManagement.md (hunk 1 above)
Comment (tlylt) [inline docs/devGuide/projectManagement.md:156]:
```suggestion
```

Comment (tlylt) [reply docs/devGuide/projectManagement.md]: Removed, thanks

**Review by jonahtanjz — APPROVED** [review]
LGTM 👍

**Pull request merged**